app/src/main/java/com/example/nuberjam/ui/main/library/LibraryFragment.kt

```diff
@@ -51,4 +53,14 @@ class LibraryFragment : Fragment() {
             }
         }
     }
+
+    private fun setFavoriteItem() {
+        binding.apply {
+            favoriteItem.tvPlaylistType.text = getString(R.string.favorite)
```

Comment:
Here i change the card type to "Favorite"

Reply:
just for tvType? how about tvName?